curriculum/challenges/english/24-b1-english-for-developers/learn-present-perfect-while-talking-about-accessibility/671f456994635c6085054cbd.md

```diff
@@ -0,0 +1,92 @@
+---
+id: 671f456994635c6085054cbd
+title: Task 37
+challengeType: 22
+dashedName: task-37
+---
+
+<!-- (Audio) James: Yes. For some reason, It hasn't been displaying correctly for some users. The captions are misaligned and out of sync, which isn't good for accessibility. -->
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Yes. For some reason, It hasn't been displaying BLANK for some BLANK. The captions are misaligned and out of sync, which isn't good for accessibility.`
+
+## --blanks--
+
+`correctly`
+
+### --feedback--
+
+This adverb describes the manner in which something should function, meaning `in the right way`.
+
+---
+
+`users`
+
+### --feedback--
+
+This noun in plural form refers to people who interact with a product or service, like software or a platform.
+
+# --explanation--
+
+`Correctly` refers to doing something in the right or intended way. For example,
+
+- `Make sure the software updates correctly to avoid errors.` - It emphasizes that the update should happen without issues or mistakes.
```

Comment:
```suggestion
`Make sure the software updates correctly to avoid errors.` - It emphasizes that the update should happen without issues or mistakes.
```